test/discrete_test.json

```diff
@@ -1376,6 +1376,33 @@
     }
   ], 
   [
+    "Poisson(0.0)", 
```

Comment:
The JSON file is generated automatically, and it will be overwritten next time the generation script is run to update the file.